style.css

```diff
@@ -22,3 +22,68 @@ body {
 }
 
 /* Add styles here! */
+
+/* Typography */
+h1 {
+  font-size: 2rem;
+  margin-block: 32px;
+}
+
+h2 {
+  font-size: 1.5rem;
+  margin-block: 24px;
+}
+
+p {
+  font-size: 1rem;
+  line-height: 1.5;
+  margin-block: 16px;
+}
+
+/* Layout */
+header {
+  padding-block: 64px;
+}
+
+main {
+  padding-block: 64px;
+  border-top: 8px solid var(--color-gray-medium);
+  background-color: var(--color-gray-light);
+}
+
+.max-width-wrapper {
+  max-width: 768px;
+  margin-inline: auto;
+}
+
+/* Elements */
+.intro-chunk {
+  max-width: 380px;
+  padding-inline: 24px;
+}
+
+.intro-chunk strong {
+  color: var(--color-primary);
+}
+
+.intro-chunk p:last-child {
+  margin-block-end: 0;
+}
+
+.card {
+  padding: 24px;
+  background-color: var(--color-white);
+  border-bottom: 8px solid var(--color-gray-medium);
+}
+
+.indented-heading {
+  padding: 16px 32px;
+  margin: -16px 0 0 -32px;
+  max-width: fit-content;
+  background-color: var(--color-secondary);
+  border-bottom: 8px solid var(--color-secondary-dark);
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💡 thought:** ‏With logical property, it would be `border-block-end: 8px solid var(--color-secondary-dark);`
https://developer.mozilla.org/en-US/docs/Web/CSS/border-block-end